### DISCUSSION TITLE
Render content in server api?

### DISCUSSION THREAD

**Discussion by fimion** [nuxt/content#1177] [Q&A]
Hey Y'all,

I am very close to making an RSS feed for my blog, but I cannot figure out how to render the content from `serverQueryContent`  in a `/server/routes` function. I think I need to use `@vue/server-renderer` but maybe there is a magic way to do this in nuxt I am not aware of?

Thanks!

**Comment by nozomuikuta** [comment]
@fimion himself found a solution for this issue.

See https://github.com/nuxt-community/feed-module/issues/106#issuecomment-1189123603
